js/jquery_validation.js

```diff
@@ -0,0 +1,285 @@
+// jQuery validation
+$(document).ready(function()
+{
+	validation();	
+});
+
+/**
+* To start validation
+*
+* @access public
+* @param  void
+* @return void
+*/
+function validation()
+{
+	$('#sign_up_form').submit(function()
+	{
+		var return_val_name = true;
+		text_elements = $('.text_field');
+		$.each(text_elements, function(index, value)
```

Comment:
Its always good habit to declare variables with var.
like var text_elements  = $('.text_field');